lib/fiware-orion-pep.js

```diff
@@ -190,6 +179,86 @@ function startProxy(callback) {
 }
 
 /**
+ * Initializes the administration server. It fills the proxyObj with information about the started proxy and passes
+ * it along.
+ *
+ * @param {Object} proxyObj         Running data of the whole proxy application.
+ */
+function initializeAdmin(proxyObj, callback) {
+    proxyObj.administration = express();
+
+    proxyObj.administration.set('port', config.resource.proxy.adminPort);
+    proxyObj.administration.set('host', '0.0.0.0');
+    proxyObj.administration.use(express.json());
+    proxyObj.administration.use(express.urlencoded());
+    proxyObj.administration.use(domainMiddleware);
+    proxyObj.administration.use(handleError);
+
+    proxyObj.administration.use(proxyObj.administration.router);
+    adminMiddleware.loadContextRoutes(proxyObj.administration);
+
+    if (config.ssl.active) {
+        var sslOptions = {
+            key: fs.readFileSync('./' + config.ssl.keyFile),
+            cert: fs.readFileSync('./' + config.ssl.certFile)
+        };
+
+        proxyObj.adminServer = https.createServer(sslOptions, proxyObj.administration);
+    } else {
+        proxyObj.adminServer = http.createServer(proxyObj.administration);
+    }
+
+    proxyObj.adminServer.listen(
+        proxyObj.administration.get('port'),
+        proxyObj.administration.get('host'), function startServer(error) {
+
+        if (error) {
+            logger.error('[PROXY-GEN-003] Error initializing administration server: ' + error.message);
```

Comment:
It is documented in the operations.md file, but just for the case of the proxy (is the same kind of error, so I don't think a second error type is needed). I've updated the operations.md information to add the alternative log message in a182244.